server/model/config.go

```diff
@@ -1010,8 +1010,8 @@ func (s *ExperimentalSettings) SetDefaults() {
 		s.EnableRemoteClusterService = NewBool(false)
 	}
 
-	if s.EnableAppBar == nil {
-		s.EnableAppBar = NewBool(false)
+	if s.DisableAppBar == nil {
+		s.DisableAppBar = NewBool(false)
```

Comment:
This is done like this since simply changing the "default" value of `EnableAppBar` would not affect existing servers, so we need ot introduce a different config setting.